src/cls/IPM/Storage/Module.cls

```diff
@@ -209,7 +211,11 @@ ClassMethod ExecutePhases(pModuleName As %String, pPhases As %List, pIsComplete
 	Try {
 		Set tVerbose = $Get(pParams("Verbose"),0)
 		Set tModule = ..NameOpen(pModuleName,,.tSC)
-		If '$isobject(tModule) {
+		If '$Isobject(tModule) {
+			Set tModule = ..ExternalNameOpen(pModuleName,,.tSC)
+			Set pModuleName = tModule.Name
```

Comment:
This would throw an undefined error when accessing Name property if the module doesn't exist. So need to move the $IsObject check to be inside the if block.